opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -91,8 +91,18 @@ def __init__(
             type, "opentelemetry.sdk.metrics.view.Aggregation"
         ] = None,
     ) -> None:
-        self._preferred_temporality = preferred_temporality
-        self._preferred_aggregation = preferred_aggregation
+        self.__preferred_temporality = preferred_temporality
+        self.__preferred_aggregation = preferred_aggregation
+
+    @property
+    def _preferred_temporality(self) -> Dict[type, AggregationTemporality]:
```

Comment:
If this still needs to be read, why not just leave the dictionary as `self._preferred_temporality`? What's the benefit of the getter here?

Reply:
It is better to have a `property` here even if it only returns an attribute because if we ever mock an instance of a `MetricExporter` subclass, the attribute will be there when we use [autospec](https://docs.python.org/3/library/unittest.mock.html#auto-speccing). I also like to make the class API very obvious (the methods it provides, including properties).